### PR TITLE
Add functional test for info

### PR DESCRIPTION
**Pull request by karthikprabhuvinod**
Add info tests

Signed-off-by: Karthik Prabhu Vinod <karthik.prabhu.vinod@intel.com>

### CODE REVIEW

File: test/functional/info/info-basic.bats [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
```
Comment (castulo) [inline test/functional/info/info-basic.bats]: you don't need to change anything in this PR, but for future reference, if you don't define any test_setup() a default one is run by testlib, and the default one looks like the one you created, it just creates a simple test environment. Same with the test_teardown(), so you could have gotten away with not adding them to this test.

File: test/functional/info/info-basic.bats [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	assert_status_is "$SWUPD_OK"
+	expected_output=$(cat <<-EOM
+		Distribution:      Swupd Test Distro
+		Installed version: 10 \(format 1\)
+		Version URL:       .*web-dir
+		Content URL:       .*web-dir
+	EOM
+	)
+
+	assert_regex_is_output "$expected_output"
```
Comment (castulo) [inline test/functional/info/info-basic.bats]: avoid using regex to check output when possible, in this case you can just do:
```
	expected_output=$(cat <<-EOM
		Distribution:      Swupd Test Distro
		Installed version: 10 (format 1)
		Version URL:       file://$TEST_DIRNAME/web-dir
		Content URL:       file://$TEST_DIRNAME/web-dir
	EOM
	)

	assert_is_output "$expected_output"
```
I think it looks much cleaner and provides better output when there are failures.

Comment (karthikprabhuvinod) [reply test/functional/info/info-basic.bats]: Ya agreed. Thanks will change

File: test/functional/info/info-basic.bats (hunk 1 above)
Comment (karthikprabhuvinod) [reply test/functional/info/info-basic.bats]: I need a to create an environment with a format other than staging. Looks like the teardown wasn't needed.  But will keep it for clarity.

Comment (castulo) [reply test/functional/info/info-basic.bats]: ahh I see, then yes, you needed that 😄

**Pull request merged**